test/test_missforest.py

```diff
@@ -0,0 +1,42 @@
+
+
+from  fancyimpute import MissForest
+
+from low_rank_data import XY, XY_incomplete, missing_mask
+#from low_rank_data import XY_categorical, XY_incomplete_categorical, missing_mask_categorical
+#from low_rank_data import XY_mixed, XY_incomplete_mixed, missing_mask_mixed
+
+from common import reconstruction_error
+
+def test_missforest_all_continous():
+    solver = MissForest(n_estimators=500)
+    XY_completed = solver.fit_transform(XY_incomplete) #dont need to normalization
+    _, missing_mae = reconstruction_error(
+        XY,
+        XY_completed,
+        missing_mask,
+        name="MissForest")
+    assert missing_mae < 0.3, "Error too high!"

```

Comment:
Every other test has this as `missing_mae < 0.1` which suggests this is too high.

Reply:
Ok, thanks